tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java

```diff
@@ -218,7 +218,7 @@ public String toString() {
    * @return the outputs of the function
    */
   public Map<String, Operand<?>> call(Scope scope, Map<String, Operand<?>> arguments) {
-    List<Operand<?>> inputList = new ArrayList<>(signature.inputNames().size());
+    List<Operand<?>> inputList = new ArrayList<>(signature().inputNames().size());
```

Comment:
I've looked and didn't see any reason for calling `signature()` instead of `signature`, since there are no class extending `ConcreteFunction` that I'm aware of. I've marked the class as `final` to make sure of that.